### PR TITLE
Add missed python requirements to the step image

### LINKED ISSUE #2583: xgboost model training fails in playground
env: http://47.114.161.156:8000/

file: `housing-explain.ipynb`

```
%%sqlflow
SELECT * FROM boston.train
TO TRAIN xgboost.gbtree
WITH
    objective="reg:squarederror",
    train.num_boost_round = 30
COLUMN crim, zn, indus, chas, nox, rm, age, dis, rad, tax, ptratio, b, lstat
LABEL medv
INTO sqlflow_models.my_xgb_regression_model;
```

image

### PR DESCRIPTION
**Pull request by shendiaomo**
Fix #2583

### CODE REVIEW

File: docker/step/Dockerfile [hunk 1]
```diff
@@ -31,7 +31,8 @@ RUN /bin/bash -c 'source find_fastest_resources.sh \
     shap==0.30.1 \
     plotille==3.7 \
     seaborn==0.9.0 \
-    dill==0.3.0"
+    dill==0.3.0 \
+	sklearn2pmml==0.56.0"
```
Comment (lhw362950217) [inline docker/step/Dockerfile]: please refine indent

Comment (shendiaomo) [reply docker/step/Dockerfile]: Done.

File: docker/step/Dockerfile [hunk 2]
```diff
@@ -31,7 +31,8 @@ RUN /bin/bash -c 'source find_fastest_resources.sh \
     shap==0.30.1 \
     plotille==3.7 \
     seaborn==0.9.0 \
-    dill==0.3.0"
+    dill==0.3.0 \
+    sklearn2pmml==0.56.0"
```
Comment (Yancey0623) [inline docker/step/Dockerfile:35]: The `sqlflow_models` wheel package has been distributed on `./build` as the following, can we just `pip install build/*.whl`?

https://github.com/sql-machine-learning/sqlflow/blob/754beefac16e4acfe1e5ea13bdc02ef27338d002/docker/dev/build.sh#L74-L76

Comment (shendiaomo) [reply docker/step/Dockerfile:35]: Yes，@lhw362950217 's PR would fix the problem.

**Review by Yancey0623 — APPROVED** [review]

**Pull request merged**